include/extractor/compressed_edge_container.hpp

```diff
@@ -35,11 +45,16 @@ class CompressedEdgeContainer
     void
     AddUncompressedEdge(const EdgeID edgei_id, const NodeID target_node, const EdgeWeight weight);
 
+    void InitializeBothwayVector();
+    unsigned ZipEdges(const unsigned f_edge_pos, const unsigned r_edge_pos);
```

Comment:
documentation, what are we zipping, how are we doing it and what kind of access impacts does that have?
Also, is this related to the main goal of the compressed geometry deduplication? Seems like an unrelated addition to me at first glance.